File: nodejs/produtor.js
const amqp = require('amqplib');

const main = async () => {
  const conn = await amqp.connect('amqp://localhost');
  const channel = await conn.createChannel();

  const queue = 'hello';
  let msg = 'Hello world';

  channel.assertQueue(queue, { durable: false });
  channel.sendToQueue(queue, Buffer.from(msg));
  console.log(" [x] Sent %s", msg);

  setTimeout(() => { 
    conn.close(); 
    process.exit(0) 
  }, 500);    
};

main();

File: python/produtor.py
import pika

RABBITMQ_HOST = 'localhost'

params     = pika.ConnectionParameters(RABBITMQ_HOST)
connection = pika.BlockingConnection()
channel    = connection.channel()

channel.queue_declare(queue='hello')

channel.basic_publish(exchange='',
                      routing_key='hello',
                      body='Hello World!')
print("[x] Enviado 'Hello World!'")

connection.close()